docs/pages/app-signing/security.mdx

```diff
@@ -39,6 +39,24 @@ You will not be able to update your app on Google Play. You may want to download
 
 Expo tools never ask you to provide your Google account credentials.
 
+## Android submit credentials
+
+### Google Service Account Key
+
+Google Service Account Key is the authentication method required to be used to submit an Android app to the Google Play Store using the EAS Submit service.
+The Google Service Account Key is permanently stored on the Expo servers (unless deleted by a developer) while being encrypted using [KMS](https://cloud.google.com/security/products/security-key-management) when at rest.
+
+#### Consequences if compromised
+
+If a malicious actor were to somehow gain access to the Google Service Account Key, they would be able to perform actions in the Google Play Console on your behalf. The actions they could perform would be limited to the permissions granted to the service account key.
+They would also be able to submit an existing app signed by your build credentials to the Google Play Store.
+When it comes to submitting an existing app to the Google Play Store, if an attacker doesn't have access to your build credentials, they won't be able to do much, as they would only be able to submit the app signed with them.
+The actor also wouldn't be able to submit a new app to the Google Play Store in your name, as the first Google Play submission needs to be done manually.
```

Comment:
```suggestion
If a malicious actor somehow gains access to your Google Service Account Key, they would be able to perform actions in the Google Play Console on your behalf. The actions they could perform would be limited to the permissions granted to the service account key. They would also be able to submit an existing app signed by your build credentials to the Google Play Store.

If an attacker doesn't have access to your build credentials, they won't be able to do much when it comes to submitting an existing app to the Google Play Store. They would only be able to submit the app signed with them. The actor also wouldn't be able to submit a new app to the Google Play Store in your name, as the first Google Play submission needs to be done manually.
```

I think there are two different parts to explain in this section. One is someone gaining access to Google Service Account Key, and the second is submitting them to Google Play Store and what happens if the attacker doesn't have access to the build credentials? Let's divide them into different paragraphs to improve the readability.